src/tracks/SampleTrack.cpp

```diff
@@ -280,6 +280,10 @@ void SampleTCO::saveSettings( QDomDocument & _doc, QDomElement & _this )
 	{
 		_this.setAttribute( "color", color().name() );
 	}
+	if( m_sampleBuffer->reversed() )
+	{
+		_this.setAttribute( "reversed", "true" );
+	}
```

Comment:
Same note about code style as above:
```suggestion
	if (m_sampleBuffer->reversed())
	{
		_this.setAttribute("reversed", "true");
	}
```

Reply:
Well I've left my opinion about these on your PR a while back xD

I could change these if you want, but I would rather keep them consistent for now (considering I used this style in the track colors PR as well). If you still want me to change these however, will do that.

Reply:
Sure, it was just a note. In the current state of the code base it's a little complicated enforcing the code style (unless it's something outrageous, which is not the case here :smile: ). I tend to write new code according to the code style though, to avoid adding to the amount of wrongly formatted code, but I'm not leaving this as a request here!

Reply:
I'd rather you start coding in the new format to get used to it now rather than later.

Reply:
I already code like this in any project I do myself, so I'm already used to it. The only reason I'm doing this is for consistency, however since there's two of you who want me to use the new style, I'll go for it over here.